src/qt/bitcoingui.cpp

```diff
@@ -325,6 +325,8 @@ void BitcoinGUI::createActions()
     optionsAction = new QAction(tr("&Options..."), this);
     optionsAction->setToolTip(tr("Modify configuration options for Gridcoin"));
     optionsAction->setMenuRole(QAction::PreferencesRole);
+    openConfigAction = new QAction(tr("Open config file"), this);
```

Comment:
```suggestion
    openConfigAction = new QAction(tr("Open config &file"), this);
```

By adding an ampersand before a character in the label, Qt will bind an <kbd>alt</kbd> key shortcut to the action. <kbd>O</kbd> and <kbd>C</kbd> are already bound to `Options...` and `Change Passphrase...` actions in this menu, so this change binds it to <kbd>F</kbd>.